Mess%20Management%20System/formClearInventory.vb

```diff
@@ -0,0 +1,23 @@
+﻿Imports MySql.Data.MySqlClient
+Public Class formClearInventory
+    Dim con As New connection 'Mysql Connection used within the Form
+    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
+        Try
+
+
+
+            con.connect()
+            Dim query As String = "set sql_safe_updates = 0 ;update mess_store set QTY = 0;"
+            Dim comm As New MySqlCommand(query, con.conn)
+            comm.ExecuteNonQuery()
+            MsgBox("Inventory Cleared")
+
+
+
+        Catch ex As Exception
+            MsgBox("Error Occured ! Please contact SSMS Tech Team !!")
+            MsgBox(ex.Message.ToString)
+
```

Comment:
Good that you have added a try-catch here but this error handling message actually does not help much. Not even for us developers while debugging the problem. I am not picking on this specific message, but most of the messages all over the code are like this.
I think a simple but related message in MsgBox would be much better here.

For a better way for future, I suggest that we should add a logging interface (to log to a log file), where not only the exceptions but all the operations are also logged with their Timestamp so that it becomes simpler to debug stuff (and all the other MsgBox msgs also can be changed to be simple but related).

It would be great if you could look into it.